src/linagora.esn.videoconference.calendar/app/components/calendar-videoconference-form/calendar-videoconference.less

```diff
@@ -13,3 +13,40 @@
     flex-grow: 1;
   }
 }
+
+.conference-btns {
+  margin-left: 10px;
+  min-width: 0;
+}
+
+.btn-row {
+  display: flex;
+
+  a {
+    display: block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  i {
+    padding-right: 15px;
+  }
+
+  .btn-link {
+    margin-left: 15px;
+  }
+}
+
+@media screen and (max-width: 990px) {
+  .conference-btns {
+    margin-left: 60px;
```

Comment:
In the mobile design, I used `justify-content: flex-end` so that the buttons are aligned from right to left. Fixing a constant space like this might not work well for various mobile devices.

Reply:
`justify-content: flex-end` have undesired effects like:
![image](https://user-images.githubusercontent.com/6764881/94273346-31645100-ff3c-11ea-9fc4-e6977dad78f9.png)


I used the left margin only for devices with width between 480px ( xs ) and 992px ( md ) so that the buttons are aligned with the input fields like in the issue.

![image](https://user-images.githubusercontent.com/6764881/94273635-90c26100-ff3c-11ea-8833-2282542d4db0.png)